cypress/integration/toynet/first.spec.ts

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable no-magic-numbers */
+
+describe('The first test', () => {
+  beforeEach(() => {
+    window.sessionStorage.clear();
+  });
+
+  it('renders the correct number of elements', () => {
+    cy.viewport(1920, 1080);
+    cy.visit('http://localhost:3000');
+    cy.contains('Try it').click();
+    cy.contains(/router/i);
+    cy.contains(/r0/);
+
+    for (let i = 0; i < 5; i++) {
+      cy.get('[data-testid=plus-icon]').first().click();
+      cy.wait(500);
+    }
+
+    cy.contains(/r6/i).should('not.exist');
+  });
+});
```

Comment:
I think that we can get rid of this `describe` and move the actual test (`it` statement) to the `Emulator.spec.ts` tests.